recipes/default.rb

```diff
@@ -21,7 +21,7 @@
 
 # Start+Enable the lvmetad service on RHEL7, it is required by default
 # but not automatically started
-if node['platform_family'] == 'rhel' && node['platform_version'].to_i >= 7
+if node['platform_family'] == 'rhel' && node[platform] != 'amazon' && node['platform_version'].to_i >= 7
```

Comment:
just a note, i tried removing `if node['platform_family'] == 'rhel' && node['platform_version'].to_i >= 7`

it failed on ubuntu
https://travis-ci.org/chef-cookbooks/lvm/builds/223384356

with 

```
     ChefSpec::Error::CommandNotStubbed:
       Executing a real command is disabled. Unregistered command:
```

not sure why it's